polls/views.py

```diff
@@ -74,3 +75,18 @@ def vote(request, question_id):
                 'question': question,
                 'error_message': _("You are only allowed to vote once."),
             })
+
+
+def free_text_vote(request, question_id):
```

Comment:
Actually, please use [Django FormView](https://docs.djangoproject.com/en/1.9/ref/class-based-views/generic-editing/#formview) instead. I'll create an issue for the vote method above.